libs/coin-modules/coin-aptos/src/__tests__/network/client.test.ts

```diff
@@ -626,8 +630,152 @@ describe("Aptos API", () => {
 
       const fees = await api.estimateFees(transactionIntent);
 
+      expect(buildSimple.mock.calls[0][0]).toEqual({
+        sender: "address1",
+        data: {
+          function: "0x1::aptos_account::transfer_coins",
+          typeArguments: ["0x1::aptos_coin::AptosCoin"],
+          functionArguments: ["address2", 100n],
+        },
+        options: {
+          maxGasAmount: 200,
+          gasUnitPrice: 100,
+          expireTimestamp: Number(Math.ceil(+ledger_timestamp / 1_000_000 + 2 * 60)),
```

Comment:
Ideally you should [mock time](https://jestjs.io/docs/timer-mocks) to assert a more effectively. This could create a flaky test.

Reply:
done